src/operator/mshadow_op.h

```diff
@@ -80,6 +80,30 @@ struct tanh_grad {
   }
 };
 
+struct exp {
+  MSHADOW_XINLINE static real_t Map(real_t a) {
+    return expf(a);
+  }
+};
+
+struct exp_grad {
+  MSHADOW_XINLINE static real_t Map(real_t a) {
```

Comment:
This is the final comment. Thanks for contribution! Please add your name to https://github.com/dmlc/mxnet/blob/master/CONTRIBUTORS.md